src/Microsoft.Azure.SignalR.Common/ConnectionStringParser.cs

```diff
@@ -39,12 +42,15 @@ internal static (string endpoint, string accessKey, string version, int? port) P
             foreach (var property in properties)
             {
                 var kvp = property.Split(KeyValueSeparator, 2);
-                if (kvp.Length != 2) continue;
+                if (kvp.Length != 2)
+                {
+                    throw new ArgumentException(string.Format(InvalidFormat, property));
```

Comment:
>ArgumentException [](start = 30, length = 17)

keep the original logic instead

Reply:
Throwing exception is just a stronger check here. And connection string is provided in portal. Customers should keep the strictly correct connection string copied from portal otherwise they keep a incorrect one. So why not make the check stricter?

Reply:
keep the original logic instead. the logic is used by other parts. modify logic in another pr if necessary. not contain in this pr.